resources/facts.txt

```diff
@@ -68,3 +68,54 @@ Several ships have wrecked on Disappointment Island.
+A snail can sleep for three years.
+Barbie`s full name is Barbara Millicent Roberts.
+A shark is the only fish that can blink with both eyes.
+There are 269 steps to the top of the Leaning Tower of Pisa
+The Ramses brand condom is named after the great pharoh Ramses II who fathered over 160 children.
+The Boston University Bridge (on Commonwealth Avenue, Boston, Massachusetts) is the only place in the world where a boat can sail under a train driving under a car driving under an airplane.
+All polar bears are left-handed.
+The state of Florida is bigger than England!
+The San Francisco Cable cars are the only mobile National Monuments.
+The U.S. military operates 234 golf courses
+John Lennon`s first girlfriend was named Thelma Pickles.
+A raisin dropped in a glass of fresh champagne will bounce up and down continually from the bottom of the glass to the top.
+the bird is the word
+If Texas were a country, its GNP would be the fifth largest of any country in the world.
+Sex burns 360 calories per hour.
+Conception occurs most in the month of December.
+400-quarter pounders can be made from 1 cow.
+There are no clocks in Las Vegas gambling casinos.
+Porcupines can float in water.
+The average woman consumes 6 lbs of lipstick in her lifetime.
+100% of lottery winners do gain weight
+In space, astronauts are unable to cry, because there is no gravity and the tears won`t flow.
+If one places a tiny amount of liquor on a scorpion, it will instantly go mad and sting itself to death.
+In Vermont, the ratio of cows to people is 10:1
+The site with the highest number of women visitors between the age of 35 and 44 years old: Alka-Seltzer.com
+All 50 states are listed across the top of the Lincoln Memorial on the back of the $5 bill.
+1 in every 4 Americans has appeared someway or another on television.
+Thirty-five percent of people who use personal ads for dating are already married.
+This common everyday occurrence composed of 59% nitrogen, 21% hydrogen, and 9% dioxide is called a `fart`.
+The airplane Buddy Holly died in was the "American Pie." (Thus the name of the Don McLean song.)
+All polar bears are left-handed.
+The word "samba" means "to rub navels together."
+`Obsession` is the most popular boat name.
+The site with the highest number of women visitors between the age of 35 and 44 years old: Alka-Seltzer.com
+Women blink nearly twice as much as men.
+The only real person to ever to appear on a pez dispenser was Betsy Ross.
+There`s a systematic lull in conversation every 7 minutes.
+Elephants are the only animals that can`t jump.
+The word Pennsylvania is misspelled on the Liberty Bell.
+Peanuts actually grow underground, as opposed to nuts like walnuts, almonds, etc. that grow on trees (and are sometimes referred to as "tree nuts"). Peanuts, along with beans and peas, belong to the single plant family, Leguminosae. Legumes are edible seeds enclosed in pods.
```

Comment:
Maybe just shorten this to "Peanuts actually grow underground, and they aren't even nuts. They're legumes!"